labs/lab-2-math.md

```diff
@@ -1,2 +1,34 @@
 # Lab 2: Basic Java math
 
+### Definitions:
+
+* **Operators**: Special characters recognized by Java that perform some sort of action.
+
+### Common operators
+
+There are five basic math operators in Java:
+* **\+** \-\-\- addition
+* **\-** \-\-\- subtraction
+* **\*** \-\-\- multiplication
+* **/** \-\-\- division
+* **%** \-\-\- modulus
+
+The first four work the same as they do in math class, for the most part. The exception is division of ints. If you use division and both numbers are ints, the result will be rounded down to the nearest integer. For example, ```9 / 5``` is 1, not 1.8.
+
+An easy way to think of modulus is taking the remainder of division. For example, ```7 % 5``` is the remainder of ```7 / 5```, 2.
+
+### Exercise #1
+Go back to your Hello World program and take a look at your int and double variables. Instead of declaring (creating) them with one value, come up with an expression that uses an operator to declare the same value. Then run your program to make sure that it has the expected behavior.
+### Bonus exercise
+Use an expression with two or more operators to initailize the variables. Java has an order of operations, just like math class, so you can also use parentheses to order things.
```

Comment:
maybe should explain or encourage the user to experiment here more.

ex. I'm pretty sure in java `x = 5 + 4 * 17` comes out to be 153 where if you follow PEMDAS it would be 73 (I could be wrong though and I can't test it right now? Pretty sure it just goes left to right (the compiler doesnt magically look for the operators to resolve the line in a different order, i think, is what I'm getting at).

Parentheses do go "first" though -  I'm not saying remove this just encourage to experiment more and see if they can see/figure how it might be different than math on a piece of paper.

also suggest changing "just like math class" to "similar to real-life math"